### ISSUE TITLE
Break up load.py

### ISSUE THREAD

**Issue by tavinathanson** [hammerlab/cohorts#37]
It's a bit monstrous at the moment.

**Comment by e5c** [comment]
(Happy to take this on as part of the public-facing example usage effort)

**Comment by tavinathanson** [comment]
Sure, let us know when you're actually moving all the code around so that we can try to avoid crazy merge conflicts.

**Comment by tavinathanson** [comment]
I think a good first step would be moving `Patient`-related functions into the `Patient` object and out of the `Cohort` object. Then, (ideally in a separate PR) `Patient` can be moved to `patient.py`. Caching logic should probably end up in a `cache.py`, though I think separate PRs for each step would be best.

**Comment by arahuja** [comment]
@tavinathanson more to do here after #138? Maybe open an issue with more specifics now that `load.py` is gone?

**Comment by tavinathanson** [comment]
I'm okay to close this for now.
